Makefile

```diff
@@ -28,10 +30,11 @@ getlint:
 # Required as mockgen is running without a relative path
 export PATH := $(PWD)/bin:$(PATH)
 
+
 # Actions
 .DEFAULT_GOAL := all
 .PHONY: all
-all: build lint test generate-markdown
+all: build lint test generate-markdown pre-commit
```

Comment:
**issue:** We lint twice in this target

Reply:
I don't see how, the other targets do not contain linting.

Reply:
`pre-commit` and `lint` both lint, no?

Reply:
The pre-commit target just adds the hook to the `.git/hooks` dir, the lint script then runs only when you commit.

Reply:
Ah... that indeed makes sense. Sorry for the confusion!